Objects/TI4_Helpers/TI4_ExploreHelper.ttslua

```diff
@@ -149,6 +149,24 @@ function explorePlanet(tileGUID, planetName)
     broadcastToAll('Exploring Planet: ' .. planetName)
 end
 
+local _frontierTokenBagGuid = nil
+function getFrontierTokenBag()
+    local frontierTokenBagObject = getObjectFromGUID(_frontierTokenBagGuid)
+    if frontierTokenBagObject then
+        return frontierTokenBagObject
+    end
+
+    local frontierTokenBagName = 'Frontier Tokens Bag'
+    for _, object in ipairs(getAllObjects()) do
+        if object.tag == 'Bag' and string.match(object.getName(), '^' .. frontierTokenBagName .. '$') then
```

Comment:
Since this is a simple comparison, better to check object.getName() == frontierTokenBagName than the more expensive string.match.